### ISSUE TITLE
show a warning when importing an edition that already exist in the inventory

### ISSUE THREAD

**Issue by thewholelifetolearn** [inventaire/inventaire-client#156]
This issue reports several problems.

I scanned last week-end a batch of books so that my inventory is more complete. While doing this, I scanned books that I had already imported. Now, I am the owner of several of the same edition. That is fine because that can happen. But I cannot correct that because it is not possible to inform Inventaire that I don't own the edition any more. The second thing, is that will importing, the import tool should have warned me that those books/editions are already listed in my inventory so that I can deselect them during the import process.

To summarize, there are 2 issues:
- remove an edition from our inventory
- warn the user while importing that the edition is already declared as owned

example: https://inventaire.io/entity/isbn:9782266235600

UPDATE: It is possible to remove an item from the inventory, but the aren't clear enough

**Comment by maxlath** [comment]
it's already possible to remove an item from an inventory

* either from the visibility box
![selection_001](https://user-images.githubusercontent.com/1596934/45088461-5abda480-b109-11e8-8016-0315dea33450.png)

* or from the item view
![selection_002](https://user-images.githubusercontent.com/1596934/45088514-7e80ea80-b109-11e8-9ff3-3c021d04a665.png)

so, if that match what you where looking for, you can rename this issue to be just the second point :)

**Comment by thewholelifetolearn** [comment]
The display of those buttons aren't the best. I didn't search them at those location. I was looking for it on the edition page.  
You should also precise that it deletes **one** edition from the inventory.

**Comment by maxlath** [comment]
I opened a different issue for the delete button: #157

**Comment by maxlath** [comment]
this was brought up as an issue again:

> Stefano: "Hi, is there any mechanism for detecting duplicate items? I know it's perfectly possible to have more than one copy of the same edition, but for personal libraries this seems rather uncommon."

> me: "it's not so uncommon to have several copies, that's how someone ended-up requesting this possibility, but I agree that it should at least always ask for a confirmation that creating a duplicated item is the desired action, and that we should have an interface to easily remove the duplicated items. I think that could be part of the mass edit interface https://trello.com/c/vwMU57Ne/127-make-it-easier-to-create-browse-and-maintain-large-inventories"

**Comment by maxlath** [comment]
* [having several copies of one edition happens (*fr*)](https://twitter.com/Bouletcorp/status/1083293442053230593)

**Comment by jum-s** [comment]
can this issue be considered solved ?
current status:

- from the list importer : 
![2020-12-28_21-18](https://user-images.githubusercontent.com/5363918/103241076-07129d00-494a-11eb-9c06-b71cab3b3336.png)

- from the `/add` page:
![2020-12-28_21-19](https://user-images.githubusercontent.com/5363918/103241105-1d205d80-494a-11eb-8698-55cdcc991976.png)

**Comment by maxlath** [comment]
I guess yes

**Comment by thewholelifetolearn** [comment]
:+1: That's fine. For consistency purpose, I would write the entire text in red in the batch importer.
